### ISSUE TITLE
Can't pass the calibration

### ISSUE THREAD

**Issue by LolJohn11** [mdias/rs_asio#392]
I can hear the game audio fine, I can also hear my guitar through the game, I see the "speakers" react to the sound, but during the calibration phase of "make some noise" the game just doesn't recognize my guitar sound as an input. All I see is the text "keep going" and nothing actually changes and I get stuck on this screen, unable to proceed further.

Audio interface: Focusrite Scarlett 18i8 2nd Gen

Rocksmith.ini
```
[Audio]
EnableMicrophone=0
ExclusiveMode=1
LatencyBuffer=4
ForceDefaultPlaybackDevice=1
ForceWDM=0
ForceDirectXSink=0
DumpAudioLog=0
MaxOutputBufferSize=0
RealToneCableOnly=0
MonoToStereoChannel=0
Win32UltraLowLatencyMode=1
[Renderer.Win32]
ShowGamepadUI=0
ScreenWidth=1920
ScreenHeight=1080
Fullscreen=1
VisualQuality=3
RenderingWidth=0
RenderingHeight=0
EnablePostEffects=1
EnableShadows=1
EnableHighResScope=1
EnableDepthOfField=1
EnablePerPixelLighting=1
MsaaSamples=4
DisableBrowser=1
[Net]
UseProxy=0
[Global]
Version=1
```

RS_ASIO
```
# for "EnableWasapiOutputs" you can use -1 to have a message prompting
# to use either WASAPI or ASIO for output every time you boot the game
[Config]
EnableWasapiOutputs=0
EnableWasapiInputs=0
EnableAsio=1

[Asio]
; use ["host"] or ["custom" + BufferSizeMode=144] - somehow that's what Rocksmith demands
; from my driver on Windows 11 and setting any other buffer results in no input (only output).
; "driver" setting is useless, Focusrite ASIO panel does not allow to set a custom buffer size like "144"
; and no other buffer size works.
BufferSizeMode=custom
CustomBufferSize=192

[Asio.Output]
; switch drivers (uncomment & comment) here and further down if using ASIO Link Pro
;Driver=ASIO Link Pro
Driver=Focusrite USB ASIO
BaseChannel=0
EnableSoftwareEndpointVolumeControl=1
EnableSoftwareMasterVolumeControl=1
SoftwareMasterVolumePercent=100

[Asio.Input.0]
;Driver=ASIO Link Pro
Driver=Focusrite USB ASIO
Channel=0
EnableSoftwareEndpointVolumeControl=1
EnableSoftwareMasterVolumeControl=1
SoftwareMasterVolumePercent=100

[Asio.Input.1]
;Driver=ASIO Link Pro
Driver=Focusrite USB ASIO
Channel=1
EnableSoftwareEndpointVolumeControl=1
EnableSoftwareMasterVolumeControl=1
SoftwareMasterVolumePercent=100
```

[RS_ASIO-log](https://github.com/mdias/rs_asio/files/13193120/RS_ASIO-log.txt)

**Comment by mdias** [comment]
Sounds like you need to increase the gain on yout guitar.
If the gain is at max on the interface already you can tweak the `SoftwareMasterVolumePercent=` value on your input to do software-gain. For example `SoftwareMasterVolumePercent=200` should increase the gain by 6dB.

**Comment by LolJohn11** [comment]
When I get to calibration, what happens is the volume "needle" representing the guitar sound level starts going up slowly (like it should) for about half a second and then just either shoots to the very top or goes "limp" to the very bottom and stays there no matter what I do, and I get the "keep going" text and nothing else.

I tried setting the `SoftwareMasterVolumePercent` to 50 first, and it almost worked since I got the "needle" to move normally, but it still wasn't enough for the game to get the perfect "loudness" desired by the calibration process. Next, I tried adjusting it to 75 and it didn't work at all. Trying different settings and even going back to 50 didn't work for some reason. I'll keep testing.
